packages/react-dom/src/__tests__/ReactErrorBoundaries-test.internal.js

```diff
@@ -653,6 +657,13 @@ describe('ReactErrorBoundaries', () => {
       container,
     );
     expect(container.firstChild.textContent).toBe('Caught an error: Hello.');
+    expect(console.error).toHaveBeenCalledWith(
+      __DEV__
+        ? expect.stringMatching(
+            /^The above error occurred in the <BrokenRender> component/,
+          )
+        : new Error('Hello'),
+    );
```

Comment:
@bvaughn looks like you added the tests here, what do you think about this strategy? I copied this from elsewhere, toWarnDev doesn't seem to work for this:

```js
expect(() => {
  ReactDOM.render(
    <ErrorBoundary>
      <BrokenRender />
    </ErrorBoundary>,
    container,
  );
}).toWarnDev('The above error occurred in the <BrokenRender> component');
```

Here's the error (with added logs of what was actually logged)

<img width="1027" alt="Screen Shot 2019-06-01 at 1 31 04 AM" src="https://user-images.githubusercontent.com/2440089/58741051-f536e400-840c-11e9-9e80-66ece7a02c38.png">